test/expression-test.js

```diff
@@ -289,3 +290,95 @@ test("convertKeyToLookup", function(){
 	equal( expression.convertKeyToLookup("foo.bar"), "foo@bar" );
 
 });
+
+test("registerConverter helpers push and pull correct values", function () {
+
+	helpers.registerConverter('numberToHex', {
+		get: function(valCompute) {
+			return valCompute().toString(16)
+		}, set: function(val, valCompute) {
+			return valCompute(parseInt("0x" + val));
+		}
+	});
+
+	var data = new CanMap({
+		observeVal: 255
+	});
+	var scope = new Scope( data );
+	var parentExpression = expression.parse("numberToHex(~observeVal)",{baseMethodType: "Call"});
+	var twoWayCompute = parentExpression.value(scope, new Scope.Options({}));
+	//twoWayCompute('34');
+
+	//var renderer = stache('<input type="text" bound-attr="numberToHex(~observeVal)" />');
+
+
+	equal(twoWayCompute(), 'ff', 'Converter called');
+	twoWayCompute('7f');
+	equal(data.attr("observeVal"), 127, 'push converter called');
+});
+
+test("registerConverter helpers push and pull multiple values", function () {
+
+	helpers.registerConverter('isInList', {
+		get: function(valCompute, list) {
+			return !!~list.indexOf(valCompute());
```

Comment:
I was into it for a while, but it's confusing to a new person so I've stopped using it as much.